include/outputs

```diff
@@ -169,16 +169,17 @@ textFail(){
   colorcode="$BAD"
   while read -r excluded_item; do
     ignore_check_name="${excluded_item%%:*}" # Check name is everything up to the first :
-    # Resource value is the second field of line included in whitelist divided by :
-    resource_value=$(awk -F ":" '{print $2}' <<< $excluded_item)
+    # Resource value is what it comes after CHECK_NAME: :
+    resource_value=$(awk -F "$CHECK_NAME:" '{print $2}' <<< "${excluded_item}")
```

Comment:
This seems overly complicated. Why not just use the same method as "ignore_check_name" above? ie
`resource_value=${excluded_item#*:}`

Reply:
Setting the delimiter to the name of the check in the awk command feel iffy to me - but if it works ... meh.

Personally I prefer using native bash when you can and not relying on external commands if you get the same result. While no one writes bash scripts with performance and resource usage being their main focus, it should at least be a consideration. Extra fork/exec/etc can start to add up, especially when we are running on a CI/CD pipeline that shares resources with other jobs. But again, this is only a suggestion - feel free to ignore.